src/components/container/SeverityCalendar/SeverityCalendar.previewdata.tsx

```diff
@@ -0,0 +1,332 @@
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: startOfDay(addDays(today, -8)).toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "severityValue",
+        answers: ["severe"],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "1238745998",
+        surveyResultID: "9",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: endOfDay(addDays(today, -8)).toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "severityValue",
+        answers: ["mild"],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "10234583466",
+        surveyResultID: "10",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: today.toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "swellingDate",
+        answers: [endOfDay(addDays(today, -9)).toISOString()],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "1237459710",
+        surveyResultID: "10",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: today.toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "severityValue",
+        answers: ["mild"],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "10234583466",
+        surveyResultID: "11",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: today.toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "swellingDate",
+        answers: [startOfDay(addDays(today, -9)).toISOString()],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "1237459710",
+        surveyResultID: "11",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: today.toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "severityValue",
+        answers: ["severe"],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "12387459988",
+        surveyResultID: "12",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: endOfDay(addDays(today, -10)).toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "severityValue",
+        answers: ["severe"],
+        insertedDate: Date.now.toString()
+    },
+    {
+        id: "102345834898",
+        surveyResultID: "12",
+        surveyID: "2",
+        surveyVersion: 1,
+        surveyName: "Survey 1",
+        surveyDisplayName: "test",
+        date: endOfDay(addDays(today, -10)).toISOString(),
+        stepIdentifier: "1",
+        resultIdentifier: "swellingDate",
```

Comment:
I see a couple references to "swelling" in the sample data.  May want to rename those to "severity" as well.